keyboards/singa/readme.md

```diff
@@ -0,0 +1,15 @@
+# singa
+
+![singa](https://i.imgur.com/VVO27Tr.jpg)
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: [amnesia0287](https://github.com/amnesia0287)
```

Comment:
Make sure that each of these lines has a double space at the end, so that they format correctly.

Reply:
Will do.

Reply:
I should also point out, this was generated with the script and then just filled in, so if its supposed to be doublespaced, you should fix the template to save yourself time :P